components/footer/Footer.vue

```diff
@@ -1,177 +1,259 @@
+            Terms of Service
+          </nuxt-link> |
+          <nuxt-link :to="{ name: 'privacy' }">
+            Privacy Policy
+          </nuxt-link>
+        </p>
+      </div>
+    </div>
   </div>
 </template>
 
 <script>
 import SparcLogo from '@/components/logo/SparcLogo.vue'
-import ContactUsModal from '@/components/footer/ContactUsModal'
 
 export default {
   name: 'SparcFooter',
   components: {
-    SparcLogo,
-    ContactUsModal
-  },
-  data: () => ({
-    isContactModalVisible: false
-  })
+    SparcLogo
+  }
 }
 </script>
 
 <style scoped lang="scss">
 @import '@/assets/_variables.scss';
 
-.footer-content {
-  size: 10pt;
-}
-
 .footer {
-  padding-top: 4em;
-  padding-bottom: 1em;
-  background: #edf1fc;
+  display: flex;
+  flex-direction: row;
+  padding-top: 5rem;
+  padding-bottom: 3rem;
 
-  color: #5d6478;
-  text-transform: uppercase;
+  &__info {
+    width: 80%;
+    padding-right: 12rem;
 
-  .link-header {
-    color: #000;
-  }
+    &--logo {
+      height: 4rem;
+      padding-bottom: 1rem;
+    }
 
-  .footer-content {
-    .contact {
-      padding-right: 32px;
-      //   margin-bottom: 2em;
+    &--blurb {
+      p {
+        font-size: 16px;
+        font-weight: normal;
+        line-height: 32px;
+        color: $dark-gray;
+        padding-bottom: 2rem;
+      }
+    }
 
-      a {
-        color: #5d6478;
-        font-weight: lighter;
-        font-size: 15px;
-        text-decoration: none;
-        text-transform: none;
-        &:hover {
-          text-decoration: none;
-          font-size: 15px;
-          color: #5d6478;
-        }
+    &--social {
+      .svg-icon {
+        width: 2.2rem;
+        margin-right: 1rem;
       }
+      padding-bottom: 2rem;
+    }
 
+    &--copyright {
       p {
-        font-weight: lighter;
-        text-transform: none;
-        font-size: 15px;
-        margin: 0;
+        color: $dark-gray;
+        font-size: 14px;
+        font-weight: 500;
+        line-height: 24px;
       }
 
-      h4 {
-        margin-top: 0;
+      a {
+        color: $dark-gray;
+      }
+    }
+  }
+
+  &__links {
+    width: 50%;
+
+    &--learn {
+      h3 {
+        font-size: 16px;
         font-weight: bold;
-        text-transform: uppercase;
-        font-size: 15px;
-        color: #5d6478;
+        line-height: 32px;
+        color: $dark-gray;
+      }
+      ul {
+        list-style: none;
+        padding-left: 0;
+
+        li {
+          padding-bottom: 1rem;
+        }
+
+        a {
+          color: $dark-gray;
+          text-decoration: none;
+          &:hover {
+            text-decoration: underline;
+          }
+        }
       }
     }
 
-    .logo {
-      height: 50px;
-      width: 120px;
-      margin-top: -9px;
+    &--social-mobile {
+      display: none;
+    }
+
+    &--copyright-mobile {
+      display: none;
     }
   }
 }
 
-.link-sections {
-  display: flex;
-  flex-wrap: wrap;
-}
+@media (min-width: 320px) and (max-width: 1023px) {
```

Comment:
No need to set a `min-width`. `max-width` will give you the same results, but still make it somewhat legible for sizes smaller than 320.